src/modules/launcher/PowerLauncher/App.xaml.cs

```diff
@@ -27,8 +27,6 @@ public partial class App : IDisposable, ISingleInstanceApp
     {
         public static PublicAPIInstance API { get; private set; }
 
-        private readonly Alphabet _alphabet = new Alphabet();
```

Comment:
[clarification] I'm guessing this line was what was causing the crash right, since it uses Pinyin4DotNet?

Reply:
The following code which is called downstream during initialization:  `PinyinHelper.toHanyuPinyinStringArray('T', _pinyinFormat);` in Alphabet.cs was causing issue.

Reply:
You can also check the full call stack in the issue related to this PR.